samples/client/petstore/elixir/lib/openapi_petstore/api/fake.ex

```diff
@@ -187,7 +187,7 @@ defmodule OpenapiPetstore.Api.Fake do
   - `{:ok, float()}` on success
   - `{:error, Tesla.Env.t}` on failure
   """
-  @spec fake_outer_number_serialize(Tesla.Env.client, keyword()) :: {:ok, float()} | {:error, Tesla.Env.t}
+  @spec fake_outer_number_serialize(Tesla.Env.client, keyword()) :: {:ok, Float} | {:error, Tesla.Env.t}
```

Comment:
`Float` should be `float()`

https://hexdocs.pm/elixir/1.15.0/typespecs.html#built-in-types